model/TanhParameters.schema

```diff
@@ -0,0 +1,11 @@
+{
+    "type" : "object",
+    "parameters" : {
+        "scale": {"type" : "integer", "minimum": 0},
```

Comment:
scale is real >0

Reply:
after a second thought maybe negative values also make sense 🤔  @eMaerthin ?

Reply:
Is scale greater than 0 or greater or equal than 0?

Reply:
scale of `0`might be difficult in numerics, but after a second thought maybe negative values also make sense 🤔  
Maybe let's make it larger than 0 and real

Reply:
agree, scale is > 0 and real